### PR TITLE
#5790 Add delayed render to `PanelBody` loading indicator

### PR DESCRIPTION
**Pull request by BLoe**
## What does this PR do?

- Closes #5790 

- [x] This PR requires a node/npm version update: let the team know on #engineering
  - `shadow-dom-testing-library`

## Checklist

- [x] Add tests
- [x] Designate a primary reviewer - @twschiller

### CODE REVIEW

File: src/sidebar/PanelBody.tsx [hunk 1]
```diff
@@ -219,7 +220,11 @@ const PanelBody: React.FunctionComponent<{
     //   );
     // }
 
-    return <Loader />;
+    return (
+      <DelayedRender millis={600}>
```
Comment (BLoe) [inline src/sidebar/PanelBody.tsx:224]: This value of 600ms is calibrated against the *AI Prompt Manager* sidebar mod, working up by increments of 100ms until the brick loading indicators stopped flashing.

Comment (twschiller) [reply src/sidebar/PanelBody.tsx:224]: This would be good as a code comment so we have the history of how the number was picked

**Comment by twschiller** [comment]
@BLoe this appears to do the hard delay but no fade-in? What's your view on a fade-in? Did you find it not necessary in this case?

**Comment by BLoe** [comment]
@twschiller I'll add the fade-in, didn't realize the delayed render doesn't handle that

**Comment by twschiller** [comment]
@BLoe as discussed - we can punt on the fade in animation for now b/c it requires more work (b/c animations appear to be currently broken in sidebar)

**Review by grahamlangford — APPROVED** [review]

**Pull request merged**